### DISCUSSION TITLE
support the endpoints required by the oauth2.0 authentication server

### DISCUSSION THREAD

**Discussion by fzucxl** [micronaut-projects/micronaut-security#1672] [Ideas]
Whether there is a plan to support the endpoints(eg:/oauth2/authorize、/oauth2/token、oauth2/refresh_token describe in https://datatracker.ietf.org/doc/html/rfc6749#section-4.1 ) required by the oauth2.0 authentication server  in a future version or not?

**Comment by sdelamo** [comment]
Micronaut Security is not a OAuth 2.0 Authorization server.
